### DISCUSSION TITLE
Forward node unable to send monitor traffic back to manager

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#6510] [Unsupported Versions]
so I've recently installed a remote sensor and it doesn't appear to be sending the monitor traffic back to the manager. It reports fine in Grid but shows 0 EPS, grafana is reporting stats (reporting 15Mb/s ish of traffic on the monitor interface) back so it can communicate it's just not sending the traffic back by the looks of it. 

I've had a look at the logs and filebeat.log is showing 
`2021-12-10T12:57:13.329Z        ERROR   [publisher_pipeline_output]     pipeline/output.go:154  Failed to connect to backoff(async(tcp://(Managername):5644)): dial tcp (manager-internal-ip):5644: connect: no route to host` 
I've checked the hosts file and conf files etc and any ref of the manager points to our external and we have firewall rules in place on the receiving end to route the traffic to the manager and the remote location has no restrictions on outbound so I'm a little lost as to where it's failing to route.

**Comment by dougburks** [comment]
From the forward node, can you ping the manager? Can you connect to the necessary ports on the manager using something like `nc`?

Did filebeat log that entry once or does it continually log that?

Have you verified that both manager and forward node are syncing their time using the same NTP server?

If you go to the Hunt interface, are you able to find any logs from the forward node at all?

**Reply by Hammer-Bear** [reply]
so i can ping the manager's external address as this is a sensor outside on a client site, we have rules on our network firewall to allow the ports stated in https://docs.securityonion.net/en/2.3/firewall.html and also a DNAT rule to direct it to the manager and we have also added the site external address as a minion under host firewall. we get traffic like Grid, grafana etc but no alerts.

If I netcat from the sensor to the manager I get a response of `Ncat: Connection reset by peer`

They both run the same NTP which is Google's NTP servers

nothing in the hunt interface using the observer name

One thing that I'm not sure if it impacts but in the logs its trying to reach its internal address and not its external though I'm not sure where to change it in filebeat.

**Reply by Hammer-Bear** [reply]
So had another look at this as it's still happening the only thing I can see which might be causing the issue is that it's trying to point to the manager's internal address and not external. however, I can't see where to update it on the sensor as salt just overwrites anything I input.
